plane/source/docs/apms-failsafe-function.rst

```diff
@@ -204,37 +243,27 @@ setting.
    <td>1</td>
    <td>ReturnToLaunch</td>
    </tr>
+   <tr>
+   <td>2</td>
+   <td>FBWA Glide</td>
+   </tr>
+   <tr>
+   <td>1</td>
+   <td>Deploy Parachute</td>
+   </tr>
    </tbody>
    </table>
 
-Failsafe battery voltage (Plane:FS_BATT_VOLTAGE)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Battery voltage to trigger failsafe. Set to 0 to disable battery voltage
-failsafe. If the battery voltage drops below this voltage then the plane
-will RTL
 
--  Units: Volts
+GCS failsafe enable (:ref:`FS_GCS_ENABL<FS_GCS_ENABL>` )
+--------------------------------------------------------
 
-Failsafe battery milliAmpHours (Plane:FS_BATT_MAH)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Battery capacity remaining to trigger failsafe. Set to 0 to disable
-battery remaining failsafe. If the battery remaining drops below this
-level then the plane will RTL
+Enable ground control station telemetry failsafe. Failsafe will trigger
+after :ref:`FS_SHORT_TIMEOUT<FS_SHORT_TIMEOUT>` and/or :ref:`FS_LONG_TIMEOUT<FS_LONG_TIMEOUT>` seconds of no MAVLink heartbeat messages.
```

Comment:
Not sure it's worth mentioning here, but MAVLink RC overrides (IE joystick from GCS), counts as a heartbeat as well as far as this timer is concerned.

Reply:
Can you have RC overrides from a GCS when you are not getting any heartbeats from the GCS?

Reply:
Yeah, your heartbeats might be lost but the overrides could still be getting through (if intermittently).

Reply:
fixed